logic/config.js

```diff
@@ -35,6 +35,7 @@ const DEFAULT_ADVANCED_SETTINGS = {
   maxmempool: 300,
   mempoolexpiry: 336,
   persistmempool: true,
+  coinstatsindex: true,
```

Comment:
```suggestion
  coinstatsindex: false,
```

This should be `false` according to [Knots default settings](https://github.com/bitcoinknots/bitcoin/blob/5f8256608fc41c7b72a01a54e9a06ba509a61c8f/src/index/coinstatsindex.h#L17)